code/model/DNEnvironment.php

```diff
@@ -392,8 +392,10 @@ public function canDeploy($member = null) {
 		}
 		// Must be logged in to check permissions
 
-		if(Permission::checkMember($member, 'ADMIN')) {
-			return true;
+		if ($this->Usage==='Production' || $this->Usage==='Unspecified') {
```

Comment:
fair enough, Just worried that a misspelling of 'Production' would break this quite easily